apicompat.go

```diff
@@ -24,6 +24,11 @@ import (
 // go/build understands this as "."
 const cwd = "."
 
+var (
+	// ErrSkipPackage is returned by the parser when a package should be skipped.
+	ErrSkipPackage = errors.New("Skipping package")
```

Comment:
Should we leave it unexported, the function that uses it isn't exported, although I haven't had a good look just yet, I don't think other libraries would need access to it